nova/scheduler/turbonomic_scheduler.py

```diff
@@ -0,0 +1,282 @@
+# Copyright (c) 2014 Rackspace Hosting
```

Comment:
Please create a separate pull request for the Turbonomic scheduler.
The first pull request should contain the request to merge the Pike branch to the Turbonomic Repo.
Then we can create a pull request with the scheduler.